src/ServicePulse.Host/vue/src/components/configuration/EndpointConnection.vue

```diff
@@ -55,8 +49,8 @@ endpointConfiguration.ConnectToServicePlatform(servicePlatformConnection);
   loading.value = false;
 }
 
-onMounted(() => {
-  getCode();
+onMounted(async () => {
+  await getCode();
```

Comment:
getCode can be made non-async since it's not doing anything async, then this can revert to what it was

Reply:
It is calling async code in line 32

Reply:
sorry, monday morning didn't notice the collapsed code